### PR TITLE
Removed extra IVT from `Microsoft.AspNet.Razor`.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
- Added tests to validate `IsStringProperty` on `TagHelperAttributeDescriptor`s.

#464

### CODE REVIEW

File: test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs [hunk 1]
```diff
@@ -717,9 +717,7 @@ public void CreateDescriptors_BuildsDescriptorsWithInheritedProperties()
         {
             // Arrange
             var errorSink = new ErrorSink();
-
-            // Also confirm isStringProperty is calculated correctly.
```
Comment (dougbu) [inline test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs]: side note: My earlier concern was around "losing" explicit checks of all `TagHelperDescriptor` properties against our expectations. That's covered now (slightly less explicitly but that's :ok:) between the test descriptor comparers and the new Razor unit tests. Note as well that without the descriptor comparer fixes, we weren't actually checking `IsStringProperty` at all :frowning:

**Comment by dougbu** [comment]
Looks as if an earlier commit broke `AttributeBlockChunkGeneratorTest.GetHashCode_ReturnsSameValue_WhenEqual` on Mono. Should probably get fix for that in first.

**Comment by dougbu** [comment]
:shipit:

**Comment by NTaylorMullen** [comment]
Talked in person about https://github.com/aspnet/Razor/pull/465#issuecomment-125729471. (CI server hasn't built the fix yet which is contained in `aspnet/Common`)

**Pull request merged**